Website/src/rendering/src/temp/GraphQLIntrospectionResult.json

```diff
@@ -1240,6 +1240,11 @@
             "name": "RichTextField",
             "ofType": null
           },
+          {
```

Comment:
Is this file required? I am confused by the folder name "temp".

Reply:
Yes, this file is being used by JSS GraphQL client. You can read about it here: https://jss.sitecore.com/docs/nextjs/graphql/introspection